sqllineage/models.py

```diff
@@ -197,6 +198,14 @@ def of(token: Token):
                     # alias without AS
                     kw_idx, _ = token.token_next_by(i=Identifier)
                 idx, _ = token.token_prev(kw_idx, skip_cm=True)
+                if idx is None:
```

Comment:
This is also a bit hacky

Reply:
Yeah looks like this is due to https://github.com/andialbrecht/sqlparse/issues/394 . I fixed both cases now by running `sqlparse.format` with `strip_comments=True` as a preprocessing step in `runner`. Do you see any issue with this?

Reply:
That's a good one!